### PR TITLE
Inheart 149:  Add FooterMenu component

### PR DESCRIPTION
**Pull request by BeataKula**
Add FooterMenu component 
Add FooterMenu to storybook
image (7)

Add FooterMenu to AuctionsPage
image (6)

https://user-images.githubusercontent.com/101329582/199966832-cdbeffc7-0e13-4c4a-8ecd-0f1f8229cd59.mp4

### CODE REVIEW

**Comment by KonradKarlinski** [comment]
Add description and images to this PR

- [ ] Add description and images to this PR

File: libs/ui/organisms/src/lib/FooterMenu/__tests__/FooterMenu.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { render } from '@inheartive/ui/testing';
+import { FooterMenu, IconNameType } from '../FooterMenu';
+
+const testID = 'FooterMenuId';
+const quantity = 1000;
+
+describe('FooterMenu', () => {
+  it('should render successfully footer menu', () => {
+    const { getByText, getByTestId } = render(
+      <FooterMenu testID={testID} activeIcon={'search'} onChange={() => console.log('FooterMenu - test')} />
```
Comment (pawelroguski) [inline libs/ui/organisms/src/lib/FooterMenu/__tests__/FooterMenu.spec.tsx:10]: I think console.log in this line should be removed

File: libs/ui/organisms/src/lib/FooterMenu/__stories__/FooterMenu.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+import { storiesOf } from '@storybook/react-native';
+import { FooterMenu, IconNameType } from '../FooterMenu';
+
+storiesOf('FooterMenu', module)
+  .addDecorator((story) => <NativeBaseProvider>{story()}</NativeBaseProvider>)
+  .add('Basic', () => {
+    const iconName = 'homepage';
+
+    const onChange = function (iconName: IconNameType): void {
+      console.log('onChangeStoriesOfFooterMenu: ' + iconName);
```
Comment (pawelroguski) [inline libs/ui/organisms/src/lib/FooterMenu/__stories__/FooterMenu.stories.tsx:12]: I think console.log in this line should be removed

File: apps/mobile/src/app/components/Auctions/AuctionsPage.tsx [hunk 3]
```diff
@@ -37,16 +41,26 @@ export function AuctionsPage() {
...
+          selectedCategoryID={selectedCategoryID}
+          onCategoryChange={(id) => setSelectedCategoryID(id)}
+          sortBy={sortBy}
+          onSortByChange={(sortBy) => setSortBy(sortBy)}
+          sortDir={sortDir}
+          onSortDirChange={(sortDir) => setSortDir(sortDir)}
+          auctions={auctions}
+        />
+      </ScrollView>
+      <FooterMenu
```
Comment (pawelroguski) [inline apps/mobile/src/app/components/Auctions/AuctionsPage.tsx:57]: We should use **page -> template** approach, but in this case i couldn't find any template for FooterMenu (or something similar)

**Review by pawelroguski — COMMENTED** [review]
Please take a look at my comments ;)

File: libs/ui/organisms/src/lib/FooterMenu/FooterMenu.tsx [hunk 4]
```diff
@@ -0,0 +1,43 @@
+/* eslint-disable @nrwl/nx/enforce-module-boundaries */
+
+import React from 'react';
+import HeartsCredits from '../../../../molecules/src/lib/HeartsCredits';
+import { colors } from 'libs/ui/theme/src/colors';
+import { Icon, IconType, Pressable } from '@inheartive/ui/atoms';
+import { Box, HStack } from 'native-base';
+
+export type IconNameType = 'homepage' | 'search' | 'heart' | 'plus' | 'star';
```
Comment (konradhanus) [inline libs/ui/organisms/src/lib/FooterMenu/FooterMenu.tsx:9]: - [ ] - convert to enum, 
- [ ] - export to separate file

**Review by konradhanus — APPROVED** [review]

**Comment by konradhanus** [comment]
reviewed by @konradhanus 
tasks left to be completed

File: apps/mobile/src/app/components/Auctions/AuctionsPage.tsx (hunk 3 above)
Comment (KonradKarlinski) [reply apps/mobile/src/app/components/Auctions/AuctionsPage.tsx:57]: FooterMenu can be only as organism. After that you can add this to auctionPageTemplate.

**Review by KonradKarlinski — APPROVED** [review]
It's look great for me. My suggestion is to create new task to stick footer to the bottom of the screen.

**Pull request merged**